src/views/templates/details/hooks/useTemplateActions.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Action } from '@openshift-console/dynamic-plugin-sdk';
+
+export const useTemplateActions = (template: V1Template): Action[] => {
+  const { t } = useKubevirtTranslation();
+
+  const actions: Action[] = React.useMemo(
+    () => [
+      {
+        id: 'template-action-create-vm',
+        label: t('Create Virtual Machine'),
+        cta: () => console.log(template),
+      },
+      {
+        id: 'template-action-create-new-template',
```

Comment:
we now call it "clone template"
see: https://github.com/kubevirt-ui/kubevirt-plugin/pull/199/files#diff-f2aabfe993673642dbeb30394a3d068805d7be2d290c372f68067273da386577R27